fediplay/mastodon.py

```diff
@@ -15,8 +15,14 @@ class StreamListener(mastodon.StreamListener):
 
     def __init__(self, queue):
         self.queue = queue
+        self.users = []
+
+    def set_users(self, users):
+        self.users = users
```

Comment:
I'd like to see the user list passed into the `StreamListener` constructor instead.